test/system/170-run-userns.bats

```diff
@@ -52,3 +52,14 @@ function _require_crun() {
     run_podman 125 run --group-add keep-groups --group-add 457 $IMAGE id
     is "$output" ".*the '--group-add keep-groups' option is not allowed with any other --group-add options" "Check group leaked into container"
 }
+
+@test "podman userns=auto in config file" {
+    skip_if_rootless "the user might not have enough IDs"
+    skip_if_remote "userns=auto is set on the server"
+
+    cat > $PODMAN_TMPDIR/userns_auto.conf <<EOF
+[containers]
+userns="auto"
+EOF
+    CONTAINERS_CONF=$PODMAN_TMPDIR/userns_auto.conf run_podman 1 run $IMAGE grep -v 4294967295 /proc/self/uid_map
```

Comment:
Could you add a comment explaining the significance of the magic constant?

And, could you add `--rm` just to be polite?

Reply:
sure, added a comment.

Also added a commit to make the existing tests polite as well